_dashboards/index.md

```diff
@@ -48,9 +48,9 @@ Use the following steps to create a dashboard using sample data.
 3. From the top menu, choose **Discover** and search for a few flights.
 4. Choose **Dashboard**, **[Flights]** Global Flight Dashboard.
 
-Your dashboard will look different than the example below.
+You have now created a dashboard, which may look different than the example below.
```

Comment:
Technically, they have not **created** the dashboard, they just pulled up the sample one that is provided. So, no need for this sentence at all.

Reply:
Revised